docs/network-policy.md

```diff
@@ -236,8 +219,115 @@ ephemeral and unpredictable.
 - Rules assume the priority in which they are written. i.e. rule set at top
   takes precedence over a rule set below it.
 
+## Antrea NetworkPolicy
+
+Antrea NetworkPolicy is another Policy CRD, which is similar to the
+ClusterNetworkPolicy CRD, however its scope is limited to a Namespace.
+The purpose of introducing this CRD is to allow admins to take advantage of
+advanced NetworkPolicy features and apply them within a Namespace to
+complement the K8s NetworkPolicies. Similar to the ClusterNetworkPolicy
+resource, Antrea NetworkPolicy can also be associated with Tiers.
+
+**Note**: Antrea NetworkPolicy is currently in "Alpha" stage and is enabled
+along with Tiers and ClusterNetworkPolicy as part of the `AntreaPolicy`
+feature gate.
+
+### The Antrea NetworkPolicy resource
+
+An example Antrea NetworkPolicy might look like this:
+```yaml
+apiVersion: security.antrea.tanzu.vmware.com/v1alpha1
+kind: NetworkPolicy
+metadata:
+  name: test-anp
+  namespace: default
+spec:
+    priority: 5
+    tier: SecurityOps
+    appliedTo:
+      - podSelector:
+          matchLabels:
+            role: db
+    ingress:
+      - action: Allow
+        from:
+          - podSelector:
+              matchLabels:
+                role: frontend
+          - podSelector:
+              matchLabels:
+                role: nondb
+            namespaceSelector:
+              matchLabels:
+                role: db
+        ports:
+          - protocol: TCP
+            port: 8080
+    egress:
+      - action: Drop
+        to:
+          - ipBlock:
+              cidr: 10.0.10.0/24
+        ports:
+          - protocol: TCP
+            port: 5978
+```
+
+### Key differences from ClusterNetworkPolicy
+
+Antrea NetworkPolicy shares it's spec with ClusterNetworkPolicy. However,
+the following documents some of the key differences between the two Antrea
+Policy CRDs.
+
+- Antrea NetworkPolicy is Namespaced while ClusterNetworkPolicy operates at
+  cluster scope.
+- Unlike the `appliedTo` in a ClusterNetworkPolicy, setting a
+  `namespaceSelector` in the `appliedTo` field is forbidden.
+- `podSelector` without a `namespaceSelector`, set within a NetworkPolicy Peer
+  of any rule, selects Pods from the Namespace in which the Antrea
+  NetworkPolicy is created. This behavior is similar to the K8s NetworkPolicy.
+
+## Antrea Policy ordering based on priorities
+
+Antrea Policy CRDs are ordered based on priorities set at various levels.
+
+### Ordering based on Tier priority
+
+With the introduction of tiers, Antrea Policies, like ClusterNetworkPolicies,
+are first enforced based on the tier to which they are associated with. i.e.
```

Comment:
associated with -> associated (as already have "**to** which")?

Reply:
done